common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
```

Comment:
As a more serious point, what would happen if `Breakpoint` was modelled in such a way that `minWidth` was not optional, and `Mobile` was modelled with 0? The pattern matching would still work with Zeroes, and I think this would do away with 'some of the Some' stuff, and a bit of extra mapping to.